src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -26,6 +27,59 @@ public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne no
             return null;
         }
 
+        public FileData TransformFileDataVersionOne(FileDataVersionOne node)
+        {
+            FileData fileData = null;
+
+            if (node != null)
+            {
+                fileData = new FileData
+                {
+                    Length = node.Length,
+                    MimeType = node.MimeType,
+                    Offset = node.Offset,
+                    ParentKey = node.ParentKey,
+                    Properties = node.Properties                    
+                };
+
+                if (fileData.Properties == null)
+                {
+                    fileData.Properties = new Dictionary<string, SerializedPropertyInfo>();
+                }
+
+                if (node.Uri != null)
+                {
+                    fileData.FileLocation = new FileLocation
+                    {
+                        Uri = node.Uri,
+                        UriBaseId = node.UriBaseId
+                    };
+                }
+
+                if (!string.IsNullOrWhiteSpace(node.Contents))
```

Comment:
At first glance, I'd ask "why shouldn't I copy the contents even if they're all whitespace?". Perhaps the answer is that all-whitespace is never a valid MIME Base64 string.

Check on that. If all-whitespace _is_ valid, change this to a simple `null` check. If not, add a comment explaining why all-whitespace is bad.

Reply:
It return empty string

Reply:
Do you mean that the MIME Base64 encoding of any all-whitespace string is the empty string?

Reply:
That's correct